src/item.cpp

```diff
@@ -3388,7 +3412,12 @@ std::string item::tname( unsigned int quantity, bool with_prefix, unsigned int t
         }
     }
     if( !faults.empty() ) {
-        damtext.insert( 0, _( "faulty " ) );
+        if( ( item::has_fault( fault_gun_blackpowder ) || item::has_fault( fault_gun_dirt ) ) &&
+            faults.size() == 1 ) {
+            damtext.insert( 0, _( dirt_symbol ) );
```

Comment:
`_()` here does nothing, since you haven't marked the corresponding string literals for translation.